### PR TITLE
docs: added notebook with openllmetry example

### PR DESCRIPTION
**Pull request by nirga**
Added a notebook example for OpenLLMetry

### CODE REVIEW

File: docs/module_guides/observability/observability.md [hunk 1]
```diff
@@ -123,6 +123,15 @@ from traceloop.sdk import Traceloop
 Traceloop.init()
 ```
 
+#### Guides
+
+```{toctree}
+---
+maxdepth: 1
+---
+/examples/callbacks/WandbCallbackHandler.ipynb
```
Comment (hatianzhang) [inline docs/module_guides/observability/observability.md:132]: is it the right link?

Comment (nirga) [reply docs/module_guides/observability/observability.md:132]: oops

File: docs/examples/callbacks/OpenLLMetry.ipynb [hunk 2]
```diff
@@ -0,0 +1,572 @@
+{
```
Comment (hatianzhang) [inline docs/examples/callbacks/OpenLLMetry.ipynb:1]: <p>can just remove those logs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Review by hatianzhang — APPROVED** [review]
LGTM

**Pull request merged**